### PR TITLE
Improve aggregate stock consumption

### LINKED ISSUE #5668: Aggregate consumption is confusing
The aggregate consumption of stock module is confusing.  Here are some frequently asked questions:

1. What does the quantity represent?  Is it the quantity at the beginning of the month, or the sum of all distributions during the month?
2. How does one model the days of stock out during the time period?

I propose that we separate the "inital quantity" and display the "quantity delivered during the month".  This will help users understand why you can have consumption during a month that you started without any stock in the depot.

### PR DESCRIPTION
**Pull request by lomamech**
- Displaying quantity at the beginning of the month, and the sum of
  stocks exits and stocks entries

closes #5668

### CODE REVIEW

**Comment by lomamech** [comment]
about the option to limit the date in relation to the chosen period it may be desirable to improve the bhDatePicker component in another issue

File: server/controllers/stock/index.js [hunk 1]
```diff
@@ -909,6 +910,129 @@ async function listLotsDepot(req, res, next) {
...
+    month_average_consumption : req.session.stock_settings.month_average_consumption,
+    average_consumption_algo : req.session.stock_settings.average_consumption_algo,
+    min_delay : req.session.stock_settings.min_delay,
+    default_purchase_interval : req.session.stock_settings.default_purchase_interval,
+    check_user_id : params.check_user_id,
+  };
+
+  try {
+    const sqlGetMonthlyStockEntry = `
+      SELECT BUID(l.inventory_uuid) AS inventory_uuid, BUID(sm.lot_uuid) AS lot_uuid,
```
Comment (jniles) [inline server/controllers/stock/index.js:951]: I'm going to try to combine these SQL queries for efficiency.

File: client/src/modules/stock/aggregated_consumption/aggregated_consumption.js [hunk 2]
```diff
@@ -145,6 +150,11 @@ function StockAggregatedConsumptionController(
       displayName : 'TABLE.COLUMNS.CONSUMPTION',
       headerCellFilter : 'translate',
       cellTemplate : 'modules/stock/aggregated_consumption/templates/lot_aggregate.tmpl.html',
+    }, {
```
Comment (jniles) [inline client/src/modules/stock/aggregated_consumption/aggregated_consumption.js:159]: These look good!  I'm going to put `text-right` on them so they can be summed easily in a user's head.

**Review by jniles — COMMENTED** [review]
These are good additions.  The UI is a bit cluttered now, but we need these values displayed.  We can make the UX nicer later.

**Pull request merged**